packages/modules/vehicles/ovms/api.py

```diff
@@ -169,21 +169,22 @@ async def _fetch_soc(self,
         if float(self.range) > 1000.0:
             self.range = str(float(self.range) / 10)
 
+        self.kms = float(statusDict['odometer']) / 10
+        self.vehicle12v = statusDict['vehicle12v']
         self.soc_ts = statusDict['m_msgtime_s']
-        log.debug("soc=" + str(self.soc) + ", range=" + str(self.range) + ", soc_ts=" + str(self.soc_ts))
 
-        return int(float(self.soc)), float(self.range), self.soc_ts
+        return int(float(self.soc)), float(self.range), self.soc_ts, float(self.kms), float(self.vehicle12v)
 
 
 # sync function
-def fetch_soc(conf: OVMS, vehicle: int) -> Union[int, float, str]:
+def fetch_soc(conf: OVMS, vehicle: int) -> Union[int, float, str, float, float]:
```

Comment:
```suggestion
def fetch_soc(conf: OVMS, vehicle: int) -> Union[int, float, str]:
```
Darauf wird nicht mehr zugegriffen.